tests/core/creators/detached.js

```diff
@@ -0,0 +1,155 @@
+/* bender-tags: editor, feature, 4461 */
+/* bender-ckeditor-plugins: wysiwygarea */
+
+( function() {
+	'use strict';
+
+	var tests = {
+		'test editor is created immediately on not detached element even with delay config': function() {
+			var editorElement = CKEDITOR.document.getById( 'editor1' ),
+				editor = CKEDITOR.replace( editorElement, {
+					delayIfDetached: true,
+					delayIfDetached_callback: function() {}
+				} );
+
+			assert.isNotNull( editor );
```

Comment:
We should add a failure message to each assert - without them:
1) it's almost impossible to track down an error on CI (like now);
2) it takes more time to understand what is wrong when checking the test after some time.

Reply:
It's `CKEDITOR.warn`stubbed test... ;(